bin/docserv

```diff
@@ -687,6 +687,14 @@ class ThreadedRESTServer(ThreadingMixIn, HTTPServer):
         logger.info("Starting HTTP server on %s:%i" % server_address)
         self.docserv = docserv
 
+def print_help():
+    print("""This is a deamon and should be invoked with the command:
+
+> systemctl start docserv CONFIG_FILENAME
```

Comment:
I'm not an expert in systemd, but it looks a bit unusual to me. Should configuration files reside under `/etc/sysconfig/`?

Reply:
No, configuration files for daemons usually reside in /etc/[PROGRAM]/[CONFIG], for example /etc/apache2/httpd.conf, /etc/postfix/main.cf, etc.

Reply:
And passing a parameters, for example config file name with @ is common practice.

Reply:
No, I forgot to write it.